tests/integration-tests/tests/dcv/test_dcv.py

```diff
@@ -45,15 +45,21 @@ def test_dcv_configuration(
     # dcv connect show url
     env = operating_system.environ.copy()
     env["AWS_DEFAULT_REGION"] = region
-    # Disable StrictHostKeyChecking by adding option to end of file
-    operating_system.system(r"echo '    StrictHostKeyChecking no' >> /etc/ssh/ssh_config")
-    result = run_command(["pcluster", "dcv", "connect", cluster.name, "--show-url"], env=env)
+
+    # add ssh key to global known hosts file to avoid ssh keychecking prompt
+    host_keys_file = operating_system.path.expanduser("~/.ssh/known_hosts")
+    add_keys_to_known_hosts(cluster.master_ip, host_keys_file)
+
+    try:
+        result = run_command(["pcluster", "dcv", "connect", cluster.name, "--show-url"], env=env)
+    finally:
+        # remove ssh key from global known hosts file
```

Comment:
same thing here: should this comment be changed to reflect the fact that the user known hosts file is now being used?